### PR TITLE
Omar tests and funct

### PR DESCRIPTION
**Pull request by OmarAbu**

### CODE REVIEW

**Review by kmaule15 — APPROVED** [review]

**Review by BrandonBuckner — APPROVED** [review]
Looks good

File: README.md [hunk 1]
```diff
@@ -66,12 +66,12 @@ git clone https://github.com/Anthony-Alagna/361Project.git
...
 
 ```shell
-cp template.env .env
+cp template..env ..env
 ```
 
 4. For the remote environment, rename the `template.env` file to `prod.env`:
 ```shell
-mv template.env prod.env
+mv template..env prod..env
```
Comment (Anthony-Alagna) [inline README.md:74]: This is also not correct

File: tests/test_create_delete_section.py
Comment (Anthony-Alagna) [inline tests/test_create_delete_section.py:1]: are these tests failing?

**Review by Anthony-Alagna — CHANGES_REQUESTED** [review]

File: tests/test_create_delete_section.py
Comment (OmarAbu) [reply tests/test_create_delete_section.py:1]: i was fixing them but then the myapp.email thing came up while iw as testing, but functionality was fine before

**Pull request closed without merging**